fromgplus/fromgplus.php

```diff
@@ -56,6 +58,9 @@ function fromgplus_addon_settings(&$a,&$s) {
 	$s .= '<label id="fromgplus-label" for="fromgplus-account">'.t('Google Account ID').' </label>';
 	$s .= '<input id="fromgplus-account" type="text" name="fromgplus-account" value="'.$account.'" />';
 	$s .= '</div><div class="clear"></div>';
+	$s .= '<label id="fromgplus-keywords-label" for="fromgplus-keywords">'.t('Add keywords to post').'</label>';
```

Comment:
With the `util/run_xgettext.sh` tool, which calls the xgettext program.

Change into the addon directory, say `addon/twitter` and call the tool with the `-s` parameter, e.g. `$>../../util/run_xgettext.sh -s` then the new messages.po file will be created in `addon/twitter/lang/C/messages.po` where Transifex can pic it up.